### PR TITLE
[HER-11]Create home page

### PR DESCRIPTION
**Pull request by andreea-dumitrascu**
## Description
Create home page

## Related
[HER-11](https://nerds-interns.atlassian.net/jira/software/projects/HER/boards/1?selectedIssue=HER-11)

### CODE REVIEW

File: src/home/carousel/styles/style-content.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  justifyContent: 'center',
+  flexDirection: 'row',
+}
+const head = {
+  textAlign: 'center',
+  padding: '50px',
+  justifyContent: 'center',
+}
+export const useStyle = makeStyles(() => ({
+  div1,
```
Comment (mihairadulescu) [inline src/home/carousel/styles/style-content.js:16]: Rename div1

**Review by mihairadulescu — CHANGES_REQUESTED** [review]

**Review by miutamihai — CHANGES_REQUESTED** [review]
Great work overall! 

The only issues I can see are name and folder structure related. I've come up with a way that looks about right, but since me leaving comments for every change I think we should make would be *tedious*, we'll have a pair programming session tomorrow specifically about that. :smile:

File: src/home/carousel/carousel-slider.jsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import { useStyle } from 'home/carousel/styles/style-carousel'
+import { useContext } from 'home/carousel/hooks/use-context'
+
+const getStyle = (index, currentIndex) => ({ ...index !== currentIndex && { display: 'none' } })
```
Comment (mihairadulescu) [inline src/home/carousel/carousel-slider.jsx:7]: do we really need ...index?

File: src/home/carousel/hooks/use-on-arrow-click.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import { useContext } from 'home/carousel/hooks/use-context'
+import { slides } from 'home/carousel/components'
+import { useActions } from 'home/carousel/hooks/use-actions'
+
+const calculateParameters = (direction, index) => {
```
Comment (mihairadulescu) [inline src/home/carousel/hooks/use-on-arrow-click.js:5]: extract in another file :)

File: src/home/carousel/styles/style-carousel.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
+import { makeStyles } from '@material-ui/core'
```
Comment (mihairadulescu) [inline src/home/carousel/styles/style-carousel.js:1]: if it`s under styles/ folder, we can drop the style- prefix.

File: src/home/carousel/styles/style-content.js [hunk 5]
```diff
@@ -0,0 +1,18 @@
+import { makeStyles } from '@material-ui/core'
```
Comment (mihairadulescu) [inline src/home/carousel/styles/style-content.js:1]: drop style- prefix from file name

**Review by mihairadulescu — CHANGES_REQUESTED** [review]

File: src/home/carousel/hooks/use-on-arrow-click.js (hunk 3 above)
Comment (andreea-dumitrascu) [reply src/home/carousel/hooks/use-on-arrow-click.js]: Done!

File: src/home/carousel/styles/style-carousel.js (hunk 4 above)
Comment (andreea-dumitrascu) [reply src/home/carousel/styles/style-carousel.js]: Done!

File: src/home/carousel/styles/style-content.js (hunk 5 above)
Comment (andreea-dumitrascu) [reply src/home/carousel/styles/style-content.js]: Done!

File: src/home/carousel/styles/style-content.js (hunk 1 above)
Comment (andreea-dumitrascu) [reply src/home/carousel/styles/style-content.js]: Done!

**Review by miutamihai — CHANGES_REQUESTED** [review]
image
Let's change the styling so that our page becomes responsive :smile:

**Review by miutamihai — APPROVED** [review]

**Review by ConstantinescuAndrei — APPROVED** [review]

**Review by Urzica-Platon-Casian — APPROVED** [review]

**Pull request merged**